userguide/virtualmachines.rst

```diff
@@ -181,8 +181,8 @@ a Virtual Machine (VM) type are described in
    |          |                    |                | number of CPUs.                                                                               |
    |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
-   | 2        | Memory Size (MiB)  | integer        | Allocate the amount of RAM in                                                                 |
-   |          |                    |                | `mebibytes <https://simple.wikipedia.org/wiki/Mebibyte>`__ for the VM.                        |
+   | 2        | Memory Size (MiB)  | integer        | Set the amount of RAM in megabytes                                                            |
+   |          |                    |                | for the VM. The available host system memory is displayed above the field.                    |
```

Comment:
"available" suggests it is available right now.  Instead, it is just the amount installed.
```suggestion
   |          |                    |                | for the VM. The host system memory is displayed above the field.                              |
```